Snapfire & Void Spirit

nikimoney opened the pull request:
- Added Snapfire and Void Spirit
- Removed some useless lines.

Numbers might need further adjusting.

nikimoney committed:
Void Spirit & Snapfire

Also cleaned up some useless lines

nikimoney committed:
Merge remote-tracking branch 'upstream/master'

nikimoney committed:
Snapfire & Void Spirit numbers

Removed a lot of useless lines as well.

EarthSalamander42 commented on game/dota_addons/barebones/scripts/npc/npc_abilities_custom.txt:
```diff
@@ -29731,4 +29731,74 @@
+
+				"Properties"
+				{
+					"MODIFIER_PROPERTY_HEALTH_REGEN_CONSTANT"	"%bonus_health_regen"
```
MODIFIER_PROPERTY_HEALTH_REGEN_CONSTANT duplicated 3 times here

nikimoney committed:
Fixed mistake

Changed the MODIFIER_PROPERTY_HEALTH_REGEN_CONSTANT duplicates to the proper modifier properties.
